compiler/aarch64/codegen/OMRInstOpCodeEnum.hpp

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
This file needs a copyright header as well, even though it is an incomplete C++ file.